src/components/views/elements/Slider.tsx

```diff
@@ -86,8 +86,8 @@ export default class Slider extends React.Component<IProps> {
             const offset = this.offset(this.props.values, this.props.value);
             selection = <div className="mx_Slider_selection">
                 <div className="mx_Slider_selectionDot" style={{left: "calc(-0.55em + " + offset + "%)"}} />
-                <hr style={{width: offset + "%"}} />
-            </div>
+                <hr style={{width: offset + "%;"}} />
```

Comment:
this looks wrong

Reply:
the indentation is right though

Reply:
Yeah, that's something else I've been trying to resolve:
![image](https://user-images.githubusercontent.com/23084468/85028712-52feb100-b173-11ea-97dc-356f79a35590.png)

Reply:
Removed for now